backend/Makefile

```diff
@@ -1,3 +1,5 @@
+IMAGE_NAME?=gitops-backend
```

Comment:
I've created a new repo for us to use:
https://quay.io/repository/redhat-appstudio/gitops-service?tab=tags

So IMAGE_NAME can now be quay.io/redhat-appstudio/gitops-service